search_test.rb

```diff
@@ -27,4 +28,13 @@ def test_basic_api_call
 			assert last_response.body.include?('90')
 			assert last_response.body.include?('https://s3.amazonaws.com/openfarm-project/production/media/pictures/attachments')
   	end
+
+    def test_all_plants_api_call
+      get '/allplants'
+      
+  		assert last_response.ok?
+    	assert  last_response.body.include?('tomato')
+  		assert last_response.body.include?('The tomato is the fruit of the tomato plant')
+  		assert last_response.body.include?('https://s3.amazonaws.com/openfarm-project/production/media/pictures/attachments')
```

Comment:
do we want to check that the research returns more than one crop?

Reply:
Added one more.